federation/pkg/kubefed/unjoin.go

```diff
@@ -72,7 +72,7 @@ func NewCmdUnjoin(f cmdutil.Factory, cmdOut, cmdErr io.Writer, config util.Admin
 
 	cmd := &cobra.Command{
 		Use:     "unjoin CLUSTER_NAME --host-cluster-context=HOST_CONTEXT",
-		Short:   "Unjoins a cluster from a federation",
+		Short:   "Unjoin a cluster from a federation",
```

Comment:
@marun updated!
@madhusudancs this one is long pending PR together with https://github.com/kubernetes/kubernetes.github.io/pull/2937.
@chenopis seems to have placed the links to these docs appropriately. This needs to be merged before the other PR can be merged.